acn-core/src/main/java/com/arrow/acn/client/model/CreateSocialEventDeviceModel.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Arrow Electronics, Inc.
```

Comment:
please correct year to 2018 for all new files.
for corrected ones - correct 2017 to 2017 - 2018 if any